src/tests/functional/plugin/shared/include/behavior/ov_executable_network/exec_network_base.hpp

```diff
@@ -150,6 +150,10 @@ TEST(OVExecutableNetworkBaseTest, smoke_LoadNetworkToDefaultDeviceNoThrow) {
 }
 
 TEST_P(OVExecutableNetworkBaseTest, canLoadCorrectNetworkToGetExecutableWithIncorrectConfig) {
+    std::set<std::string> metaPlugins = {"AUTO", "MULTI", "HETERO"};
+    if (metaPlugins.end() != std::find(metaPlugins.begin(), metaPlugins.end(), target_device)) {
+        GTEST_SKIP();
+    }
```

Comment:
Hi, just discovered that these cases have been refactored and will no longer run [PR16431](https://github.com/openvinotoolkit/openvino/pull/16413)
And for MULTI and HETERO, in this test suit, MULTI and HETERO will throw because it does not has target devices like MULTI:**.
So I think we can add a specific case for it in [PR16840](https://github.com/openvinotoolkit/openvino/pull/16840)